CMakeLists.txt

```diff
@@ -163,3 +163,55 @@ INSTALL(FILES ${CMAKE_BINARY_DIR}/include/singa/singa_config.h DESTINATION
 INSTALL (DIRECTORY ${CMAKE_BINARY_DIR}/lib DESTINATION
     ${CMAKE_INSTALL_PREFIX}
     PATTERN "*libgtest.a" EXCLUDE )
+
+IF (USE_PYTHON)
+INSTALL(DIRECTORY ${PROJECT_SOURCE_DIR}/python 
```

Comment:
can we package the files in $CMAKE_INSTALL_PREFIX/lib/python/site-package/singa/, which is the place of python files when we type `make install` and is the default folder of python modules?